services/QuillLMS/db/structure.sql

```diff
@@ -138,11 +125,11 @@ CREATE FUNCTION public.timespent_question(act_sess integer, question character v
                       'epoch' FROM (activity_sessions.completed_at - activity_sessions.started_at)
                     )
                 END) INTO time_spent FROM activity_sessions WHERE id = act_sess AND state='finished';
-
+                
```

Comment:
I know this file is auto-generated, but it's weird that it's making these empty lines with spaces in them.